### PR TITLE
Extend short document titles

### LINKED ISSUE #234: Page titles are below 30 characters
Page titles should follow the following format:

Primary Keyword - Secondary Keyword | Brand Name
So in the case of : Work | simplabs
We could have: Full-Service Engineering - Our Work | simplabs  or  Web & Mobile - Our Work | simplabs (these are just examples, we should check with Mark)

Contact Us | simplabs
Imprint | simplabs
Trainline | simplabs
Elixir | simplabs
DD-WRT | simplabs
Expedition | simplabs
About | simplabs
Services | simplabs
Team Augmentation | simplabs
Imprint | simplabs
Work | simplabs
Training | simplabs
Timify | simplabs
Ember | simplabs
Blog | simplabs
Enginification | simplabs
From SPA to PWA | simplabs
Ember.SimpleAuth | simplabs
excellent 2.0.0 | simplabs
Rails API Auth | simplabs
excellent 1.7.2 | simplabs

### PR DESCRIPTION
**Pull request by pichfl**
For better search results

Closes #234

### CODE REVIEW

File: src/ui/components/PageCaseDdWrt/template.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <Header
     @active="work"
     @title="DD-WRT NXT Case Study"
-    @documentTitle="DD-WRT NXT Case Study | Work"
+    @documentTitle="Providing powerful UI for DD-WRT NXT using Ember.js | Work"
```
Comment (marcoow) [inline src/ui/components/PageCaseDdWrt/template.hbs]:
```suggestion
    @documentTitle="A modern UI for DD-WRT NXT based on Ember.js | Work"
```

File: src/ui/components/PageCaseStudyExpedition/template.hbs [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <Header
     @active="work"
     @title="Expedition Case Study"
-    @documentTitle="Expedition Case Study | Work"
+    @documentTitle="Full-stack work for Expedition online travel magazine | Work"
```
Comment (marcoow) [inline src/ui/components/PageCaseStudyExpedition/template.hbs]:
```suggestion
    @documentTitle="An architecture based on Elixir and Ember.js for Expedition | Work"
```

File: src/ui/components/PageCaseStudyTimify/template.hbs [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <Header
     @active="work"
     @title="Timify Case Study"
-    @documentTitle="Timify Case Study | Work"
+    @documentTitle="Rebuilt MVP for long term success for Timify | Work"
```
Comment (marcoow) [inline src/ui/components/PageCaseStudyTimify/template.hbs]:
```suggestion
    @documentTitle="Rebuilding an MVP for long term success for Timify | Work"
```

File: src/ui/components/PageCaseStudyTrainline/template.hbs [hunk 4]
```diff
@@ -2,7 +2,7 @@
   <Header
     @active="work"
     @title="Trainline Case Study"
-    @documentTitle="Trainline Case Study | Work"
+    @documentTitle="Mobile web app with Ember.js for Trainline | Work"
```
Comment (marcoow) [inline src/ui/components/PageCaseStudyTrainline/template.hbs]:
```suggestion
    @documentTitle="A mobile train ticket counter based on Ember.js for Trainline | Work"
```

File: src/ui/components/PageContact/template.hbs [hunk 5]
```diff
@@ -2,6 +2,7 @@
   <Header
     @active="contact"
     @title="Contact"
+    @documentTitle="Contact us for successful projects and increased team effectiveness"
```
Comment (marcoow) [inline src/ui/components/PageContact/template.hbs]:
```suggestion
    @documentTitle="Contact us to discuss how we can help you"
```

File: src/ui/components/PageServices/template.hbs [hunk 6]
```diff
@@ -2,6 +2,7 @@
   <Header
     @active="services"
     @title="Services"
+    @documentTitle="Full-stack & tutoring & team augmentation | Our services"
```
Comment (marcoow) [inline src/ui/components/PageServices/template.hbs]:
```suggestion
    @documentTitle="Full Stack Engineering & Tutoring & Team Augmentation | Our services"
```

File: src/ui/components/PageTutoring/template.hbs [hunk 7]
```diff
@@ -2,6 +2,7 @@
   <Header
     @active="services"
     @title="Tutoring"
+    @documentTitle="Level up your team's effectiveness with our tutoring | Services"
```
Comment (marcoow) [inline src/ui/components/PageTutoring/template.hbs]:
```suggestion
    @documentTitle="Leveling up teams' confidence with Tutoring | Services"
```

File: src/ui/components/PageTeamAugmentation/template.hbs [hunk 8]
```diff
@@ -2,7 +2,7 @@
   <Header
     @active="services"
     @title="Team Augmentation"
-    @documentTitle="Team Augmentation | Services"
+    @documentTitle="Your team, more effective. | Team Augmentation | Services"
```
Comment (marcoow) [inline src/ui/components/PageTeamAugmentation/template.hbs]:
```suggestion
    @documentTitle="Boosting teams' effectiveness with Team Augmentation | Services"
```

File: src/ui/components/PageWhySimplabs/template.hbs [hunk 9]
```diff
@@ -2,6 +2,7 @@
   <Header
     @active="about"
     @title="Why simplabs"
+    @documentTitle="Ember.js consulting, Full service solutions | Why us"
```
Comment (marcoow) [inline src/ui/components/PageWhySimplabs/template.hbs]:
```suggestion
    @documentTitle="Guiding our clients from idea to release | Why simplabs"
```

**Review by marcoow — COMMENTED** [review]
I left some suggestions

File: src/ui/components/PageCaseStudyExpedition/template.hbs (hunk 2 above)
Comment (pichfl) [reply src/ui/components/PageCaseStudyExpedition/template.hbs]: This drops the "Full stack" keyword from the title, which I'd rather have in there for SEO.

File: src/ui/components/PageCaseStudyTrainline/template.hbs (hunk 4 above)
Comment (pichfl) [reply src/ui/components/PageCaseStudyTrainline/template.hbs]: `A mobile train ticket counter based on Ember.js for Trainline | Work | simplabs` is 79 chars, which could exceed the recommend limits. Also, this removes a chance for us to associate "mobile web app" with simplabs.

File: src/ui/components/PageCaseStudyExpedition/template.hbs (hunk 2 above)
Comment (marcoow) [reply src/ui/components/PageCaseStudyExpedition/template.hbs]: but instead it add _"Ember.js"_ and in particular  _"Elixir"_ which I think is **more valuable** for our SEO?

File: src/ui/components/PageCaseStudyTrainline/template.hbs (hunk 4 above)
Comment (marcoow) [reply src/ui/components/PageCaseStudyTrainline/template.hbs]: what is the recommended limit?

**Comment by pichfl** [comment]
@marcoow I know that some of these are not very elegant compared to your proposed changes, but they are a strong source for search engine keywords

File: src/ui/components/PageCaseStudyExpedition/template.hbs (hunk 2 above)
Comment (pichfl) [reply src/ui/components/PageCaseStudyExpedition/template.hbs]: Depends on where you want to go, just wanted to make sure this is an informed change 👍

File: src/ui/components/PageCaseStudyTrainline/template.hbs (hunk 4 above)
Comment (pichfl) [reply src/ui/components/PageCaseStudyTrainline/template.hbs]: > Google typically displays the first 50–60 characters of a title tag. If you keep your titles under 60 characters, our research suggests that you can expect about 90% of your titles to display properly. 

It's not an exact science, I've seen up to 70 as well.

Comment (marcoow) [reply src/ui/components/PageCaseStudyTrainline/template.hbs]: maybe _"Mobile train tickets with Ember.js for Trainline | Work"_?

**Review by marcoow — APPROVED** [review]

**Pull request merged**